xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -246,8 +247,20 @@ void CPVRGUIInfo::UpdateTimeshift(void)
 {
   if (!CServiceBroker::GetPVRManager().IsPlayingTV() && !CServiceBroker::GetPVRManager().IsPlayingRadio())
   {
+    // If nothing is playing (anymore), there is no need to poll the timeshift values from the clients.
     CSingleLock lock(m_critSection);
-    m_iStartTime = 0;
+    if (m_bHasTimeshiftData)
```

Comment:
Shouldn't this be negated?

Reply:
No. If not pledging and there is data out must be cleared. All good imo.